lab_3_ann_retriever/start.py

```diff
@@ -3,6 +3,7 @@
 """
 
 # pylint:disable=duplicate-code, too-many-locals, too-many-statements, unused-variable
+import lab_3_ann_retriever.main as m
```

Comment:
strange import

Reply:
Why? I've done this in previous labs too